MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -265,8 +265,8 @@ MeshLib::Mesh* VtkMeshConverter::constructMesh(const double* pixVal,
 	if (value_vec->empty())
 		properties.removePropertyVector("Colour");
 
-	boost::optional< MeshLib::PropertyVector<unsigned>& > materials =
-		properties.createNewPropertyVector<unsigned>("MaterialIDs", MeshLib::MeshItemType::Cell, 1);
+	boost::optional< MeshLib::PropertyVector<int>& > materials =
+		properties.createNewPropertyVector<int>("MaterialIDs", MeshLib::MeshItemType::Cell, 1);
 	materials->insert(materials->end(), elements.size(), 0);
```

Comment:
Yes, `materials` should be empty. In my opninion the values are inserted into the `std::vector` correctly. I added a check if `createNewPropertyVector` creates a valid object.